client/my-sites/email/titan-mail-add-mailboxes/index.jsx

```diff
@@ -125,10 +126,12 @@ class TitanMailAddMailboxes extends React.Component {
 		} );
 	};
 
-	handleContinue = () => {
+	handleContinue = async () => {
```

Comment:
I think this breaks the "busy" state of the button and feels a bit off - you click on the button and something is happening but there is no visual feedback